src/rules/quotemarkRule.ts

```diff
@@ -67,13 +71,15 @@ class QuoteWalker extends Lint.RuleWalker {
             const firstCharacter = text.charAt(0);
             const lastCharacter = text.charAt(text.length - 1);
 
-            if (this.quoteMark === QuoteMark.SINGLE_QUOTES) {
-                if (firstCharacter !== "'" || lastCharacter !== "'") {
-                    failure = this.createFailure(position, width, Rule.SINGLE_QUOTE_FAILURE);
-                }
-            } else if (this.quoteMark === QuoteMark.DOUBLE_QUOTES) {
-                if (firstCharacter !== "\"" || lastCharacter !== "\"") {
-                    failure = this.createFailure(position, width, Rule.DOUBLE_QUOTE_FAILURE);
+            const expectedQuoteMarkCharacter = getQuoteMarkCharacter(this.quoteMark);
```

Comment:
I thought that tslint will support single quotes, double quotes and **backticks** in the not-so-distant future.
So, I implemented it, for starting easily backticks-support.

How do you think it?